<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastDialog</class>
 <widget class="QDialog" name="ContrastDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>228</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Min intensity:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QSpinBox" name="minIntensitySpinBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="minIntensitySlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Max intensity:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="maxIntensitySlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="maxIntensitySpinBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>minIntensitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>minIntensitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minIntensitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>minIntensitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxIntensitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>maxIntensitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxIntensitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>maxIntensitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
